admin/app/components/solidus_admin/ui/forms/input/component.rb

```diff
@@ -93,13 +93,22 @@ def call
       with_content options_for_select(@attributes.delete(:choices), @attributes.delete(:value))
     end
 
-    tag.public_send(
-      @tag,
-      content,
+    options = {
       "data-controller": stimulus_id,
       "data-#{stimulus_id}-custom-validity-value": @error.presence,
       "data-action": "#{stimulus_id}#clearCustomValidity",
       **@attributes
-    )
+    }
+
+    if @tag == :input
```

Comment:
Rails 8 doesn't allow sending content to tags that don't support it, so we have to conditionally invoke with the right parameters. See: https://github.com/rails/rails/blob/8-0-stable/actionview/CHANGELOG.md#rails-800rc1-october-19-2024

Reply:
This is valid for [other tags](https://github.com/rails/rails/blob/149a0ad259b1d360c9a7d04b354bbd3fa9326452/actionview/lib/action_view/helpers/tag_helper.rb#L84-L97) as well, which we might not be using now. What about checking something more generic on the method signature? Like:

```
method(:method_name).parameters.any? { |type, name| name == :content }
```

Not so elegant but it's more generic, what do you think?

Reply:
Sure, that'll work.